scripts/build.sh

```diff
@@ -46,6 +46,24 @@ gox \
     -output "pkg/{{.OS}}_{{.Arch}}/nomad" \
     .
 
+echo ""
+if pkg-config --exists lxc; then
```

Comment:
So there should be a way to opt out of the LXC build since we will release via the Vagrant box that will have lxc, we need a way to build the binary without LXC too